pkgs/tools/networking/zrok-bin/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchzip, patchelf }:
+
+stdenv.mkDerivation rec {
+  pname = "zrok-bin";
+  version = "0.3.4";
+
+  meta = {
+    description = "Geo-scale, next-generation sharing platform built on top of OpenZiti";
+    homepage = "https://zrok.io";
+    maintainers = [ lib.maintainers.bandresen ];
+    platforms = [ "x86_64-linux" ];
+    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
+    license = lib.licenses.apsl20;
+  };
+
+  src = fetchzip {
+    name = "zrok";
+    url = "https://github.com/openziti/zrok/releases/download/v${version}/zrok_${version}_linux_amd64.tar.gz";
```

Comment:
Why not build the go program from source?

Reply:
The web part is a pita.

Reply:
Yeah, that's the part I would love mentorship on. The Go part seems to work fine, but the npm part is tricky for me. I'm not at home in that universe.